lib/3scale_toolbox.rb

```diff
@@ -1,9 +1,32 @@
+require '3scale_toolbox/configuration'
+
 module ThreeScaleToolbox
+  @config_file = nil
+  @configuration = nil
+
   def self.load_plugins
     plugin_paths.each { |plugin_path| require plugin_path }
   end
 
   def self.plugin_paths
     Gem.find_files('3scale_toolbox_plugin')
   end
+
+  def self.config_file
+    # 3SCALERC env var has priority over $HOME/.3scalerc.yaml file
+    @config_file ||= ENV['THREESCALERC']
```

Comment:
`THREESCALERC` is a pretty weird name. Why not something more meaningful like `THREESCALE_CLI_CONFIG' ?